code/modules/cm_marines/equipment/sentries.dm

```diff
@@ -19,7 +19,12 @@
 	icon_state = "sentry_case"
 	w_class = 5
 	storage_slots = 6
-	can_hold = list() //Nada. Once you take the stuff out it doesn't fit back in.
```

Comment:
Have you tested the effect of this?

Reply:
Yes, it means you can actually put stuff back in.

Reply:
What about other kind of stuff?

Reply:
Who cares? It's a box. I don't see why it should magically preclude stuff from going on.

Reply:
Alright.